src/test/java/org/opensearch/ad/task/ADTaskCacheManagerTests.java

```diff
@@ -8,46 +8,9 @@
  * Modifications Copyright OpenSearch Contributors. See
  * GitHub history for details.
  */
-
+/* @anomaly-detection.create-components. Commented until we have support for SDKClusterSettings for extensions
```

Comment:
Do we have an issue for `SDKClusterSettings`? Let's link it here

Reply:
Yeah, we need an issue for SDKClusterSettings constructor with params.  It's primarily just used for tests.

Reply:
Sure I've created the issue here : https://github.com/opensearch-project/opensearch-sdk-java/issues/503. Ill link it to each Test class that I've commented out